src/Pages/Services/BookDistribution/index.tsx

```diff
@@ -1,13 +1,18 @@
 import { ServiceDescription } from "../PageLayout/index";
-import { bookDistribution ,bookDistributionImages} from "./constant";
-
+import {
+  bookDistribution,
+  bookDistributionImages,
+  contactDetails,
+} from "./constant";
 
 export const BookDistribution: React.FC = () => {
   return (
     <ServiceDescription
       title="Book Distribution"
       content={bookDistribution}
       imageUrl={bookDistributionImages}
+      contactNumber={contactDetails}
+      emailId={[]}
```

Comment:
you don't need to pass empty emailId because it's optional

Reply:
Resolved it.